src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,59 @@
+import VenueChat from "components/molecules/VenueChat";
+import ChatsList from "components/molecules/ChatsList";
+import LiveSchedule from "components/molecules/LiveSchedule";
+import { useSelector } from "hooks/useSelector";
+import { useVenueId } from "hooks/useVenueId";
+import React, { useState } from "react";
+import { useFirestoreConnect } from "react-redux-firebase";
+import "./Sidebar.scss";
+
+const Sidebar = () => {
+  const venueId = useVenueId();
+  useFirestoreConnect({
+    collection: "users",
+    where: ["enteredVenueIds", "array-contains", venueId],
+    storeAs: "chatUsers",
+  });
+  const [tab, setTab] = useState(0);
+  const { privateChats, chatUsers } = useSelector((state) => ({
+    privateChats: state.firestore.ordered.privatechats,
+    chatUsers: state.firestore.data.chatUsers,
+  }));
```

Comment:
As above RE: extracting selectors, particularly about avoiding this 'compound selector' style

Reply:
this is still an issue

Ideal fix:

Create the selectors outside of the component (eg. top of file, `src/utils/selectors.ts`, etc):

```typescript
const privateChatsSelector = (state: RootState) => state.firestore.ordered.privatechats
const chatUsersSelector = (state: RootState) => state.firestore.data.chatUsers
```

Then you can do:
```suggestion
  const privateChats = useSelector(privateChatsSelector)
  const chatUsers = useSelector(chatUsersSelector)
```

Reply:
fixed in 70e8253242c4c56f3678a5035e0e6566912da044